docs/pages/reference/mesheryctl/mesheryctl-registry-generate.md

```diff
@@ -54,21 +54,22 @@ mesheryctl registry generate --spreadsheet-id "1DZHnzxYWOlJ69Oguz4LkRVTFM79kC2tu
 </div>
 </pre> 
 
-Generate Meshery Models and Component from csv files in a local directory.
+Easily generate Meshery models and components from CSV files in a local directory using the following command. To get started, you can use a sample template directory provided [here](https://github.com/meshery/meshery/tree/master/docs/assets/model-and-component-csv-templates)
```

Comment:
@Aijeyomah, this will be overwritten with the next time the mesheryctl documentation is auto-generated. Please use the instructions in the Meshery CLI contributing page, which offers guidance on how/where to put additional usage notes with guarantee that they won't be overwritten. 

@alphaX86, perhaps, you can assist by doing this on @Aijeyomah's behalf.

Reply:
@Aijeyomah, Any additional examples or instructions that you offer in addition to the docs from go
cobra are really helpful to users. Here is the link to the instructions that you can use to capture the details that you have provided in a way in which will be preserved and not overwritten - https://docs.meshery.io/project/contributing/contributing-cli#preserving-manually-added-documentation